### PR TITLE
Wip ann

### PR DESCRIPTION
**Pull request by AnnJohn**
Updated objectives, added overview, added content (from this [Readme](https://github.com/learn-co-curriculum/debugging-errors-readme) so that the content presented in the video is added to the text, changed "Lab" title to "Instructions"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,10 +58,49 @@ You've solved games of *Clue* with less information. This is my favorite part of
...
+
+Will result in:
+```text
+2: syntax error, unexpected end-of-input, expecting keyword_end
+```
+Here, Ruby is saying that on line 2, there is a missing `end` (every `do` keyword must be followed by some code and then an `end` keyword). Always read the full details of syntax errors and look for line numbers, which usually appear at the beginning of the error message.
+
+### TypeErrors
+
+When you try and do a mathematical operation on two objects of a different type, you will recieve a TypeError.  For example if you try and add a string to an integer, Ruby will complain.
```
Comment (deniznida) [inline README.md]: typo `recieve` should be`receive`

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+In this lab, you'll be reading error messages from tests. This lab is designed so that both running the files *and* running the test suite via the `learn` command will show the error messages for you to decode. Moving forward though, you'll be reading error messages mainly through running the test suite. 
 
 ## Objectives
 
-1. Understand how to read error messages.
-2. Practice fixing some common errors.
-3. Practice using the `learn` command in your terminal to run the tests for a lab.
-4. Practice submitting a lab. 
+1. Read the three different parts of an errors message.
+2. Identify four error types- name errors, syntax errors, types errors, and division errors- and fix them
```
Comment (deniznida) [inline README.md]: should we also add argument errors ? Don't know how relevant it is for this lecture

**Comment by deniznida** [comment]
@AnnJohn added some comments

File: README.md (hunk 2 above)
Comment (AnnJohn) [inline README.md]: not relevant for this lecture, since it's not covered in the video

**Comment by deniznida** [comment]
@AnnJohn after fixing typo, good to merge

**Pull request merged**